lightstep/resource_dashboard_test.go

```diff
@@ -155,42 +152,3 @@ resource "lightstep_dashboard" "test" {
 		},
 	})
 }
```

Comment:
ooc why are we deleting these tests?

Reply:
Oh, it's deleting test *helper functions* that are no longer called by anyone. It's not actually deleting any unit tests!